src/elements/kw-share-card/kw-share-card.html

```diff
@@ -135,7 +135,7 @@
                             <p>[[share.description]]</p>
                         </div>
                         <div class="actions">
-                            <button class="orange-button remix-button">remix<img src="/assets/icons/shares/remix.svg" /></button>
+                            <button on-tap="_onTapRemix" class="orange-button remix-button">remix<img src="/assets/icons/shares/remix.svg" /></button>
```

Comment:
@paulvarache I would suggest move the app into next ticket, due to I supposed to finish the design for that ticket, when these share card design merged, it will help me more easy to focus on app integration. Thanks.